internal/request/graphql/schema/collection.go

```diff
@@ -651,8 +767,10 @@ func astTypeToKind(
 				return client.FieldKind_NILLABLE_BOOL_ARRAY, nil
 			case typeInt:
 				return client.FieldKind_NILLABLE_INT_ARRAY, nil
-			case typeFloat:
-				return client.FieldKind_NILLABLE_FLOAT_ARRAY, nil
+			case typeFloat, typeFloat64:
+				return client.FieldKind_NILLABLE_FLOAT64_ARRAY, nil
+			case typeFloat32:
```

Comment:
todo: I think I saw a comment from Shahzad asking you to test this, but just in case it was for something else, please make sure this is tested.

Reply:
It was something else. I'll look into testing.

Reply:
I added a test that creates a schema with all supported types to ensure we hit a maximum of these.